src/main/resources/templates/earnings.html

```diff
@@ -0,0 +1,90 @@
+{# A test page for the dual pane page #}
+{% extends "base/li-sidebar-page" %}
+{% block title %} Exception Null - Earnings{% endblock %}
+
+{% block content %}
+<div class="earnings">
+	<div class="left">
+		<div class="card">
+			<div class="title">Earnings <span class="hint">this month, estimated</span></div>
+			<div class="big-number">$11.15</div>
+		</div>
+		<div class="card-spacing"></div>
+		<div class="card">
+			<div class="title">Reviews Completed <span class="hint">this month</span></div>
+			<div class="big-number">8</div>
+		</div>
+		<div class="card-spacing"></div>
+		<div class="card">
+			<div class="title">Rating <span class="hint">this month, average</span></div>
+			<div class="big-number">3.11</div>
+		</div>
+	</div>
+
+	<div class="right">
+		<div class="card">
+			<div class="title">History</div>
+			<canvas id="earning-review-history">8</canvas>
+		</div>
+	</div>
+</div>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.6.0/Chart.bundle.min.js"></script>
+<script>
+	(function() {
+		window.onload = function() {
+			let ctx = document.getElementById('earning-review-history').getContext('2d');
+			let myChart = new Chart(ctx, {
+				type: 'bar',
+				data: {
+					labels: ["May (this month)", "April", "March", "February", "January"],
```

Comment:
Can we have the order of the earnings on the chart go from January to May (chronological order)?